



insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","Insulin resistance is strongly associated with type II diabetes. 'Diabetogenic' factors including FFA, TNFalpha and cellular stress induce insulin resistance through inhibition of IRS1 functions. Serine/threonine phosphorylation, interaction with SOCS, regulation of the expression, modification of the cellular localization, and degradation represent the molecular mechanisms stimulated by them. Various kinases (ERK, JNK, IKKbeta, PKCzeta, PKCtheta and mTOR) are involved in this process.
The development of type II diabetes requires impaired beta-cell function. Chronic hyperglycemia has been shown to induce multiple defects in beta-cells. Hyperglycemia has been proposed to lead to large amounts of reactive oxygen species (ROS) in beta-cells, with subsequent damage to cellular components including PDX-1. Loss of PDX-1, a critical regulator of insulin promoter activity, has also been proposed as an important mechanism leading to beta-cell dysfunction.
Although there is little doubt as to the importance of genetic factors in type II diabetes, genetic analysis is difficult due to complex interaction among multiple susceptibility genes and between genetic and environmental factors. Genetic studies have therefore given very diverse results. Kir6.2 and IRS are two of the candidate genes. It is known that Kir6.2 and IRS play central roles in insulin secretion and insulin signal transmission, respectively.
Source: KEGG","http://www.wikipathways.org/instance/WP1584","class",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","Cell-matrix adhesions play essential roles in important biological processes including cell motility, cell proliferation, cell differentiation, regulation of gene expression and cell survival. At the cell-extracellular matrix contact points, specialized structures are formed and termed focal adhesions, where bundles of actin filaments are anchored to transmembrane receptors of the integrin family through a multi-molecular complex of junctional plaque proteins. Some of the constituents of focal adhesions participate in the structural link between membrane receptors and the actin cytoskeleton, while others are signalling molecules, including different protein kinases and phosphatases, their substrates, and various adapter proteins. Integrin signaling is dependent upon the non-receptor tyrosine kinase activities of the FAK and src proteins as well as the adaptor protein functions of FAK, src and Shc to initiate downstream signaling events. These signalling events culminate in reorganization of the actin cytoskeleton; a prerequisite for changes in cell shape and motility, and gene expression. Similar morphological alterations and modulation of gene expression are initiated by the binding of growth factors to their respective receptors, emphasizing the considerable crosstalk between adhesion- and growth factor-mediated signalling. Source: KEGG http://www.genome.jp/kegg/pathway/hsa/hsa04510.html","http://www.wikipathways.org/instance/WP306","class",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values ("prion disease","WikiPathways","Prion diseases are rare, genetic, transmissible and sporadic diseases, which are caused by mutations in the PRNP gene. This gene is located on chromosome 20p13 and is composed of two exons. Mutations in the PRNP gene cause conformational changes in the prion protein (PRNP). The normal PRNP (protein) changes into the pathologic PRNP. A molecular pathway can give a better understanding in prion diseases.

This pathway is a prion disease pathway, that describes what happens when there is a mutation in the PRNP gene. The left part of the pathway represents the pathway retrieved from literature and the right part of the pathway represents data found using databases. The left part shos the interaction of pathological prion protein with an unknown receptor protein, this interaction activates a signalling pathway. The endoplasmic reticulum releases calcium and ER stress is induced. Activation of Caspase 12 by ER-stress is followed by cleavage and activation of the executioner Caspase-3, causing neuronal apoptosis. According to the databases, NCAM-1 can initiate two mechanisms: the activation of FGFR and formation of intracellular signalling complexes. NCAM-1 interacts with Fyn and FAK, resulting in phosphorylation of these two tyrosine kinases. Phosphorylation of Fyn and FAK results in activation of MAPK, ERK1 and 2, cAMP response element binding protein (CREB) and transcription factors ELK and NFkB. CREB activates transcription of genes which are important for axonal growth, survival, and synaptic plasticity in neurons.","http://www.wikipathways.org/instance/WP3995","instance","neuron;neuron");
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","Homologous recombination, also known as general recombination, is a type of genetic recombination in which nucleotide sequences are exchanged between two similar or identical strands of DNA. Source: Wikipedia","http://www.wikipathways.org/instance/WP186","class",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","Studies in the past decade have suggested that the basic mechanism of DNA replication initiation is conserved in all kingdoms of life. Initiation in unicellular eukaryotes, in particular Saccharomyces cerevisiae (budding yeast), is well understood, and has served as a model for studies of DNA replication initiation in multicellular eukaryotes, including humans. In general terms, the first step of initiation is the binding of the replication initiator to the origin of replication. The replicative helicase is then assembled onto the origin, usually by a helicase assembly factor. Either shortly before or shortly after helicase assembly, some local unwinding of the origin of replication occurs in a region rich in adenine and thymine bases (often termed a DNA unwinding element, DUE). The unwound region provides the substrate for primer synthesis and initiation of DNA replication. The best-defined eukaryotic origins are those of S. cerevisiae, which have well-conserved sequence elements for initiator binding, DNA unwinding and binding of accessory proteins. In multicellular eukaryotes, unlike S. cerevisiae, these loci appear not to be defined by the presence of a DNA sequence motif. Indeed, choice of replication origins in a multicellular eukaryote may vary with developmental stage and tissue type. In cell-free models of metazoan DNA replication, such as the one provided by Xenopus egg extracts, there are only limited DNA sequence specificity requirements for replication initiation. Source description: http://www.reactome.org/content/detail/69306.","http://www.wikipathways.org/instance/WP466","instance",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","Oxidative phosphorylation is the process in which ATP is formed as a result of the transfer of electrons from NADH or FADH2 to O2 by a series of electron carriers. This process, which takes place in mitochondria, is the major source of ATP in aerobic organisms. [1]","http://www.wikipathways.org/instance/WP623","class",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values ("obesity","WikiPathways","Lipid metabolism is the break down or storage of fats for energy; these fats are obtained from consuming food and absorbing them or they are synthesized by an animal's liver. Lipid metabolism does exist in plants, though the processes differ in some ways when compared to animals. Lipogenesis is the process of synthesizing these fats. Lipid metabolism often begins with hydrolysis, which occurs when a chemical breaks down as a reaction to coming in contact with water. Since lipids (fats) are hydrophobic, hydrolysis in lipid metabolism occurs in the cytoplasm which ends up creating glycerol and fatty acids. Due to the hydrophobic nature of lipids they require special transport proteins known as lipoproteins, which are hydrophilic. Lipoproteins are categorized by their density levels. The varying densities between the types of lipoproteins are characteristic to what type of fats they transport. A number of these lipoproteins are synthesized in the liver, but not all of them originate from this organ. Description source: Wikipedia","http://www.wikipathways.org/instance/WP3965","class","white fat cell");
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","This pathway is a concise and simplified version of the basic proteins involved in angiogenesis. The proces is regulated by hyopoxia, which results in the transcription of eg. VEGF1, FGF2, PDGFbeta, MMP9, MMP2 and Ang1. These proteins bind to their receptors and by doing this they initiate different pathways that result in vessel formation.","http://www.wikipathways.org/instance/WP1539","instance",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","The cell cycle is the series of events that takes place in a cell leading to its division and duplication (replication). Regulation of the cell cycle involves processes crucial to the survival of a cell, including the detection and repair of genetic damage as well as the prevention of uncontrolled cell division. Two key classes of regulatory molecules, cyclins and cyclin-dependent kinases (CDKs), determine a cell's progress through the cell cycle.

Sources: Wikipedia:Cell cycle, KEGG","http://www.wikipathways.org/instance/WP179","class",NULL);




insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();
insert into pathway_entity ) values ();




insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","Insulin resistance is strongly associated with type II diabetes. 'Diabetogenic' factors including FFA, TNFalpha and cellular stress induce insulin resistance through inhibition of IRS1 functions. Serine/threonine phosphorylation, interaction with SOCS, regulation of the expression, modification of the cellular localization, and degradation represent the molecular mechanisms stimulated by them. Various kinases (ERK, JNK, IKKbeta, PKCzeta, PKCtheta and mTOR) are involved in this process.
The development of type II diabetes requires impaired beta-cell function. Chronic hyperglycemia has been shown to induce multiple defects in beta-cells. Hyperglycemia has been proposed to lead to large amounts of reactive oxygen species (ROS) in beta-cells, with subsequent damage to cellular components including PDX-1. Loss of PDX-1, a critical regulator of insulin promoter activity, has also been proposed as an important mechanism leading to beta-cell dysfunction.
Although there is little doubt as to the importance of genetic factors in type II diabetes, genetic analysis is difficult due to complex interaction among multiple susceptibility genes and between genetic and environmental factors. Genetic studies have therefore given very diverse results. Kir6.2 and IRS are two of the candidate genes. It is known that Kir6.2 and IRS play central roles in insulin secretion and insulin signal transmission, respectively.
Source: KEGG","http://www.wikipathways.org/instance/WP1584","class",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","Cell-matrix adhesions play essential roles in important biological processes including cell motility, cell proliferation, cell differentiation, regulation of gene expression and cell survival. At the cell-extracellular matrix contact points, specialized structures are formed and termed focal adhesions, where bundles of actin filaments are anchored to transmembrane receptors of the integrin family through a multi-molecular complex of junctional plaque proteins. Some of the constituents of focal adhesions participate in the structural link between membrane receptors and the actin cytoskeleton, while others are signalling molecules, including different protein kinases and phosphatases, their substrates, and various adapter proteins. Integrin signaling is dependent upon the non-receptor tyrosine kinase activities of the FAK and src proteins as well as the adaptor protein functions of FAK, src and Shc to initiate downstream signaling events. These signalling events culminate in reorganization of the actin cytoskeleton; a prerequisite for changes in cell shape and motility, and gene expression. Similar morphological alterations and modulation of gene expression are initiated by the binding of growth factors to their respective receptors, emphasizing the considerable crosstalk between adhesion- and growth factor-mediated signalling. Source: KEGG http://www.genome.jp/kegg/pathway/hsa/hsa04510.html","http://www.wikipathways.org/instance/WP306","class",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values ("prion disease","WikiPathways","Prion diseases are rare, genetic, transmissible and sporadic diseases, which are caused by mutations in the PRNP gene. This gene is located on chromosome 20p13 and is composed of two exons. Mutations in the PRNP gene cause conformational changes in the prion protein (PRNP). The normal PRNP (protein) changes into the pathologic PRNP. A molecular pathway can give a better understanding in prion diseases.

This pathway is a prion disease pathway, that describes what happens when there is a mutation in the PRNP gene. The left part of the pathway represents the pathway retrieved from literature and the right part of the pathway represents data found using databases. The left part shos the interaction of pathological prion protein with an unknown receptor protein, this interaction activates a signalling pathway. The endoplasmic reticulum releases calcium and ER stress is induced. Activation of Caspase 12 by ER-stress is followed by cleavage and activation of the executioner Caspase-3, causing neuronal apoptosis. According to the databases, NCAM-1 can initiate two mechanisms: the activation of FGFR and formation of intracellular signalling complexes. NCAM-1 interacts with Fyn and FAK, resulting in phosphorylation of these two tyrosine kinases. Phosphorylation of Fyn and FAK results in activation of MAPK, ERK1 and 2, cAMP response element binding protein (CREB) and transcription factors ELK and NFkB. CREB activates transcription of genes which are important for axonal growth, survival, and synaptic plasticity in neurons.","http://www.wikipathways.org/instance/WP3995","instance","neuron;neuron");
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","Homologous recombination, also known as general recombination, is a type of genetic recombination in which nucleotide sequences are exchanged between two similar or identical strands of DNA. Source: Wikipedia","http://www.wikipathways.org/instance/WP186","class",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","Studies in the past decade have suggested that the basic mechanism of DNA replication initiation is conserved in all kingdoms of life. Initiation in unicellular eukaryotes, in particular Saccharomyces cerevisiae (budding yeast), is well understood, and has served as a model for studies of DNA replication initiation in multicellular eukaryotes, including humans. In general terms, the first step of initiation is the binding of the replication initiator to the origin of replication. The replicative helicase is then assembled onto the origin, usually by a helicase assembly factor. Either shortly before or shortly after helicase assembly, some local unwinding of the origin of replication occurs in a region rich in adenine and thymine bases (often termed a DNA unwinding element, DUE). The unwound region provides the substrate for primer synthesis and initiation of DNA replication. The best-defined eukaryotic origins are those of S. cerevisiae, which have well-conserved sequence elements for initiator binding, DNA unwinding and binding of accessory proteins. In multicellular eukaryotes, unlike S. cerevisiae, these loci appear not to be defined by the presence of a DNA sequence motif. Indeed, choice of replication origins in a multicellular eukaryote may vary with developmental stage and tissue type. In cell-free models of metazoan DNA replication, such as the one provided by Xenopus egg extracts, there are only limited DNA sequence specificity requirements for replication initiation. Source description: http://www.reactome.org/content/detail/69306.","http://www.wikipathways.org/instance/WP466","instance",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","Oxidative phosphorylation is the process in which ATP is formed as a result of the transfer of electrons from NADH or FADH2 to O2 by a series of electron carriers. This process, which takes place in mitochondria, is the major source of ATP in aerobic organisms. [1]","http://www.wikipathways.org/instance/WP623","class",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values ("obesity","WikiPathways","Lipid metabolism is the break down or storage of fats for energy; these fats are obtained from consuming food and absorbing them or they are synthesized by an animal's liver. Lipid metabolism does exist in plants, though the processes differ in some ways when compared to animals. Lipogenesis is the process of synthesizing these fats. Lipid metabolism often begins with hydrolysis, which occurs when a chemical breaks down as a reaction to coming in contact with water. Since lipids (fats) are hydrophobic, hydrolysis in lipid metabolism occurs in the cytoplasm which ends up creating glycerol and fatty acids. Due to the hydrophobic nature of lipids they require special transport proteins known as lipoproteins, which are hydrophilic. Lipoproteins are categorized by their density levels. The varying densities between the types of lipoproteins are characteristic to what type of fats they transport. A number of these lipoproteins are synthesized in the liver, but not all of them originate from this organ. Description source: Wikipedia","http://www.wikipathways.org/instance/WP3965","class","white fat cell");
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","This pathway is a concise and simplified version of the basic proteins involved in angiogenesis. The proces is regulated by hyopoxia, which results in the transcription of eg. VEGF1, FGF2, PDGFbeta, MMP9, MMP2 and Ang1. These proteins bind to their receptors and by doing this they initiate different pathways that result in vessel formation.","http://www.wikipathways.org/instance/WP1539","instance",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","The cell cycle is the series of events that takes place in a cell leading to its division and duplication (replication). Regulation of the cell cycle involves processes crucial to the survival of a cell, including the detection and repair of genetic damage as well as the prevention of uncontrolled cell division. Two key classes of regulatory molecules, cyclins and cyclin-dependent kinases (CDKs), determine a cell's progress through the cell cycle.

Sources: Wikipedia:Cell cycle, KEGG","http://www.wikipathways.org/instance/WP179","class",NULL);




insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","Insulin resistance is strongly associated with type II diabetes. 'Diabetogenic' factors including FFA, TNFalpha and cellular stress induce insulin resistance through inhibition of IRS1 functions. Serine/threonine phosphorylation, interaction with SOCS, regulation of the expression, modification of the cellular localization, and degradation represent the molecular mechanisms stimulated by them. Various kinases (ERK, JNK, IKKbeta, PKCzeta, PKCtheta and mTOR) are involved in this process.
The development of type II diabetes requires impaired beta-cell function. Chronic hyperglycemia has been shown to induce multiple defects in beta-cells. Hyperglycemia has been proposed to lead to large amounts of reactive oxygen species (ROS) in beta-cells, with subsequent damage to cellular components including PDX-1. Loss of PDX-1, a critical regulator of insulin promoter activity, has also been proposed as an important mechanism leading to beta-cell dysfunction.
Although there is little doubt as to the importance of genetic factors in type II diabetes, genetic analysis is difficult due to complex interaction among multiple susceptibility genes and between genetic and environmental factors. Genetic studies have therefore given very diverse results. Kir6.2 and IRS are two of the candidate genes. It is known that Kir6.2 and IRS play central roles in insulin secretion and insulin signal transmission, respectively.
Source: KEGG","http://www.wikipathways.org/instance/WP1584","class",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","Cell-matrix adhesions play essential roles in important biological processes including cell motility, cell proliferation, cell differentiation, regulation of gene expression and cell survival. At the cell-extracellular matrix contact points, specialized structures are formed and termed focal adhesions, where bundles of actin filaments are anchored to transmembrane receptors of the integrin family through a multi-molecular complex of junctional plaque proteins. Some of the constituents of focal adhesions participate in the structural link between membrane receptors and the actin cytoskeleton, while others are signalling molecules, including different protein kinases and phosphatases, their substrates, and various adapter proteins. Integrin signaling is dependent upon the non-receptor tyrosine kinase activities of the FAK and src proteins as well as the adaptor protein functions of FAK, src and Shc to initiate downstream signaling events. These signalling events culminate in reorganization of the actin cytoskeleton; a prerequisite for changes in cell shape and motility, and gene expression. Similar morphological alterations and modulation of gene expression are initiated by the binding of growth factors to their respective receptors, emphasizing the considerable crosstalk between adhesion- and growth factor-mediated signalling. Source: KEGG http://www.genome.jp/kegg/pathway/hsa/hsa04510.html","http://www.wikipathways.org/instance/WP306","class",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values ("prion disease","WikiPathways","Prion diseases are rare, genetic, transmissible and sporadic diseases, which are caused by mutations in the PRNP gene. This gene is located on chromosome 20p13 and is composed of two exons. Mutations in the PRNP gene cause conformational changes in the prion protein (PRNP). The normal PRNP (protein) changes into the pathologic PRNP. A molecular pathway can give a better understanding in prion diseases.

This pathway is a prion disease pathway, that describes what happens when there is a mutation in the PRNP gene. The left part of the pathway represents the pathway retrieved from literature and the right part of the pathway represents data found using databases. The left part shos the interaction of pathological prion protein with an unknown receptor protein, this interaction activates a signalling pathway. The endoplasmic reticulum releases calcium and ER stress is induced. Activation of Caspase 12 by ER-stress is followed by cleavage and activation of the executioner Caspase-3, causing neuronal apoptosis. According to the databases, NCAM-1 can initiate two mechanisms: the activation of FGFR and formation of intracellular signalling complexes. NCAM-1 interacts with Fyn and FAK, resulting in phosphorylation of these two tyrosine kinases. Phosphorylation of Fyn and FAK results in activation of MAPK, ERK1 and 2, cAMP response element binding protein (CREB) and transcription factors ELK and NFkB. CREB activates transcription of genes which are important for axonal growth, survival, and synaptic plasticity in neurons.","http://www.wikipathways.org/instance/WP3995","instance","neuron;neuron");
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","Homologous recombination, also known as general recombination, is a type of genetic recombination in which nucleotide sequences are exchanged between two similar or identical strands of DNA. Source: Wikipedia","http://www.wikipathways.org/instance/WP186","class",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","Studies in the past decade have suggested that the basic mechanism of DNA replication initiation is conserved in all kingdoms of life. Initiation in unicellular eukaryotes, in particular Saccharomyces cerevisiae (budding yeast), is well understood, and has served as a model for studies of DNA replication initiation in multicellular eukaryotes, including humans. In general terms, the first step of initiation is the binding of the replication initiator to the origin of replication. The replicative helicase is then assembled onto the origin, usually by a helicase assembly factor. Either shortly before or shortly after helicase assembly, some local unwinding of the origin of replication occurs in a region rich in adenine and thymine bases (often termed a DNA unwinding element, DUE). The unwound region provides the substrate for primer synthesis and initiation of DNA replication. The best-defined eukaryotic origins are those of S. cerevisiae, which have well-conserved sequence elements for initiator binding, DNA unwinding and binding of accessory proteins. In multicellular eukaryotes, unlike S. cerevisiae, these loci appear not to be defined by the presence of a DNA sequence motif. Indeed, choice of replication origins in a multicellular eukaryote may vary with developmental stage and tissue type. In cell-free models of metazoan DNA replication, such as the one provided by Xenopus egg extracts, there are only limited DNA sequence specificity requirements for replication initiation. Source description: http://www.reactome.org/content/detail/69306.","http://www.wikipathways.org/instance/WP466","instance",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","Oxidative phosphorylation is the process in which ATP is formed as a result of the transfer of electrons from NADH or FADH2 to O2 by a series of electron carriers. This process, which takes place in mitochondria, is the major source of ATP in aerobic organisms. [1]","http://www.wikipathways.org/instance/WP623","class",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values ("obesity","WikiPathways","Lipid metabolism is the break down or storage of fats for energy; these fats are obtained from consuming food and absorbing them or they are synthesized by an animal's liver. Lipid metabolism does exist in plants, though the processes differ in some ways when compared to animals. Lipogenesis is the process of synthesizing these fats. Lipid metabolism often begins with hydrolysis, which occurs when a chemical breaks down as a reaction to coming in contact with water. Since lipids (fats) are hydrophobic, hydrolysis in lipid metabolism occurs in the cytoplasm which ends up creating glycerol and fatty acids. Due to the hydrophobic nature of lipids they require special transport proteins known as lipoproteins, which are hydrophilic. Lipoproteins are categorized by their density levels. The varying densities between the types of lipoproteins are characteristic to what type of fats they transport. A number of these lipoproteins are synthesized in the liver, but not all of them originate from this organ. Description source: Wikipedia","http://www.wikipathways.org/instance/WP3965","class","white fat cell");
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","This pathway is a concise and simplified version of the basic proteins involved in angiogenesis. The proces is regulated by hyopoxia, which results in the transcription of eg. VEGF1, FGF2, PDGFbeta, MMP9, MMP2 and Ang1. These proteins bind to their receptors and by doing this they initiate different pathways that result in vessel formation.","http://www.wikipathways.org/instance/WP1539","instance",NULL);
insert into pathway_entity (Disease,source,dms_description,link,type,Cell Type) values (NULL,"WikiPathways","The cell cycle is the series of events that takes place in a cell leading to its division and duplication (replication). Regulation of the cell cycle involves processes crucial to the survival of a cell, including the detection and repair of genetic damage as well as the prevention of uncontrolled cell division. Two key classes of regulatory molecules, cyclins and cyclin-dependent kinases (CDKs), determine a cell's progress through the cell cycle.

Sources: Wikipedia:Cell cycle, KEGG","http://www.wikipathways.org/instance/WP179","class",NULL);




insert into pathway_entity (Disease,source,dms_description,link,type,cell_type,PMOID,pathway_name) values (NULL,"WikiPathways","Insulin resistance is strongly associated with type II diabetes. 'Diabetogenic' factors including FFA, TNFalpha and cellular stress induce insulin resistance through inhibition of IRS1 functions. Serine/threonine phosphorylation, interaction with SOCS, regulation of the expression, modification of the cellular localization, and degradation represent the molecular mechanisms stimulated by them. Various kinases (ERK, JNK, IKKbeta, PKCzeta, PKCtheta and mTOR) are involved in this process.
The development of type II diabetes requires impaired beta-cell function. Chronic hyperglycemia has been shown to induce multiple defects in beta-cells. Hyperglycemia has been proposed to lead to large amounts of reactive oxygen species (ROS) in beta-cells, with subsequent damage to cellular components including PDX-1. Loss of PDX-1, a critical regulator of insulin promoter activity, has also been proposed as an important mechanism leading to beta-cell dysfunction.
Although there is little doubt as to the importance of genetic factors in type II diabetes, genetic analysis is difficult due to complex interaction among multiple susceptibility genes and between genetic and environmental factors. Genetic studies have therefore given very diverse results. Kir6.2 and IRS are two of the candidate genes. It is known that Kir6.2 and IRS play central roles in insulin secretion and insulin signal transmission, respectively.
Source: KEGG","http://www.wikipathways.org/instance/WP1584","class",NULL,"PMO:00006370","Type II diabetes mellitus");
insert into pathway_entity (Disease,source,dms_description,link,type,cell_type,PMOID,pathway_name) values (NULL,"WikiPathways","Cell-matrix adhesions play essential roles in important biological processes including cell motility, cell proliferation, cell differentiation, regulation of gene expression and cell survival. At the cell-extracellular matrix contact points, specialized structures are formed and termed focal adhesions, where bundles of actin filaments are anchored to transmembrane receptors of the integrin family through a multi-molecular complex of junctional plaque proteins. Some of the constituents of focal adhesions participate in the structural link between membrane receptors and the actin cytoskeleton, while others are signalling molecules, including different protein kinases and phosphatases, their substrates, and various adapter proteins. Integrin signaling is dependent upon the non-receptor tyrosine kinase activities of the FAK and src proteins as well as the adaptor protein functions of FAK, src and Shc to initiate downstream signaling events. These signalling events culminate in reorganization of the actin cytoskeleton; a prerequisite for changes in cell shape and motility, and gene expression. Similar morphological alterations and modulation of gene expression are initiated by the binding of growth factors to their respective receptors, emphasizing the considerable crosstalk between adhesion- and growth factor-mediated signalling. Source: KEGG http://www.genome.jp/kegg/pathway/hsa/hsa04510.html","http://www.wikipathways.org/instance/WP306","class",NULL,"PMO:00014404","Focal Adhesion");
insert into pathway_entity (Disease,source,dms_description,link,type,cell_type,PMOID,pathway_name) values ("prion disease","WikiPathways","Prion diseases are rare, genetic, transmissible and sporadic diseases, which are caused by mutations in the PRNP gene. This gene is located on chromosome 20p13 and is composed of two exons. Mutations in the PRNP gene cause conformational changes in the prion protein (PRNP). The normal PRNP (protein) changes into the pathologic PRNP. A molecular pathway can give a better understanding in prion diseases.

This pathway is a prion disease pathway, that describes what happens when there is a mutation in the PRNP gene. The left part of the pathway represents the pathway retrieved from literature and the right part of the pathway represents data found using databases. The left part shos the interaction of pathological prion protein with an unknown receptor protein, this interaction activates a signalling pathway. The endoplasmic reticulum releases calcium and ER stress is induced. Activation of Caspase 12 by ER-stress is followed by cleavage and activation of the executioner Caspase-3, causing neuronal apoptosis. According to the databases, NCAM-1 can initiate two mechanisms: the activation of FGFR and formation of intracellular signalling complexes. NCAM-1 interacts with Fyn and FAK, resulting in phosphorylation of these two tyrosine kinases. Phosphorylation of Fyn and FAK results in activation of MAPK, ERK1 and 2, cAMP response element binding protein (CREB) and transcription factors ELK and NFkB. CREB activates transcription of genes which are important for axonal growth, survival, and synaptic plasticity in neurons.","http://www.wikipathways.org/instance/WP3995","instance",NULL,"PMO:00014472","Prion disease pathway");
insert into pathway_entity (Disease,source,dms_description,link,type,cell_type,PMOID,pathway_name) values (NULL,"WikiPathways","Homologous recombination, also known as general recombination, is a type of genetic recombination in which nucleotide sequences are exchanged between two similar or identical strands of DNA. Source: Wikipedia","http://www.wikipathways.org/instance/WP186","class",NULL,"PMO:00014484","Homologous recombination");
insert into pathway_entity (Disease,source,dms_description,link,type,cell_type,PMOID,pathway_name) values (NULL,"WikiPathways","Studies in the past decade have suggested that the basic mechanism of DNA replication initiation is conserved in all kingdoms of life. Initiation in unicellular eukaryotes, in particular Saccharomyces cerevisiae (budding yeast), is well understood, and has served as a model for studies of DNA replication initiation in multicellular eukaryotes, including humans. In general terms, the first step of initiation is the binding of the replication initiator to the origin of replication. The replicative helicase is then assembled onto the origin, usually by a helicase assembly factor. Either shortly before or shortly after helicase assembly, some local unwinding of the origin of replication occurs in a region rich in adenine and thymine bases (often termed a DNA unwinding element, DUE). The unwound region provides the substrate for primer synthesis and initiation of DNA replication. The best-defined eukaryotic origins are those of S. cerevisiae, which have well-conserved sequence elements for initiator binding, DNA unwinding and binding of accessory proteins. In multicellular eukaryotes, unlike S. cerevisiae, these loci appear not to be defined by the presence of a DNA sequence motif. Indeed, choice of replication origins in a multicellular eukaryote may vary with developmental stage and tissue type. In cell-free models of metazoan DNA replication, such as the one provided by Xenopus egg extracts, there are only limited DNA sequence specificity requirements for replication initiation. Source description: http://www.reactome.org/content/detail/69306.","http://www.wikipathways.org/instance/WP466","instance",NULL,"PMO:00014497","DNA Replication");
insert into pathway_entity (Disease,source,dms_description,link,type,cell_type,PMOID,pathway_name) values (NULL,"WikiPathways","Oxidative phosphorylation is the process in which ATP is formed as a result of the transfer of electrons from NADH or FADH2 to O2 by a series of electron carriers. This process, which takes place in mitochondria, is the major source of ATP in aerobic organisms. [1]","http://www.wikipathways.org/instance/WP623","class",NULL,"PMO:00014609","Oxidative phosphorylation");
insert into pathway_entity (Disease,source,dms_description,link,type,cell_type,PMOID,pathway_name) values ("obesity","WikiPathways","Lipid metabolism is the break down or storage of fats for energy; these fats are obtained from consuming food and absorbing them or they are synthesized by an animal's liver. Lipid metabolism does exist in plants, though the processes differ in some ways when compared to animals. Lipogenesis is the process of synthesizing these fats. Lipid metabolism often begins with hydrolysis, which occurs when a chemical breaks down as a reaction to coming in contact with water. Since lipids (fats) are hydrophobic, hydrolysis in lipid metabolism occurs in the cytoplasm which ends up creating glycerol and fatty acids. Due to the hydrophobic nature of lipids they require special transport proteins known as lipoproteins, which are hydrophilic. Lipoproteins are categorized by their density levels. The varying densities between the types of lipoproteins are characteristic to what type of fats they transport. A number of these lipoproteins are synthesized in the liver, but not all of them originate from this organ. Description source: Wikipedia","http://www.wikipathways.org/instance/WP3965","class",NULL,"PMO:00014625","Lipid Metabolism Pathway");
insert into pathway_entity (Disease,source,dms_description,link,type,cell_type,PMOID,pathway_name) values (NULL,"WikiPathways","This pathway is a concise and simplified version of the basic proteins involved in angiogenesis. The proces is regulated by hyopoxia, which results in the transcription of eg. VEGF1, FGF2, PDGFbeta, MMP9, MMP2 and Ang1. These proteins bind to their receptors and by doing this they initiate different pathways that result in vessel formation.","http://www.wikipathways.org/instance/WP1539","instance",NULL,"PMO:00014675","Angiogenesis");
insert into pathway_entity (Disease,source,dms_description,link,type,cell_type,PMOID,pathway_name) values (NULL,"WikiPathways","The cell cycle is the series of events that takes place in a cell leading to its division and duplication (replication). Regulation of the cell cycle involves processes crucial to the survival of a cell, including the detection and repair of genetic damage as well as the prevention of uncontrolled cell division. Two key classes of regulatory molecules, cyclins and cyclin-dependent kinases (CDKs), determine a cell's progress through the cell cycle.

Sources: Wikipedia:Cell cycle, KEGG","http://www.wikipathways.org/instance/WP179","class",NULL,"PMO:00014709","Cell Cycle");
